### PR TITLE
Add form validation

### PR DESCRIPTION
**Pull request by aakbarkhan**
-The content of the email field has to be in lower case.
-You should implement the following interactions:
-When the user submits the form, you check if the email is in lower case. (check)
-If the validation is OK, the form is sent. (checked)
-If the validation is not OK, you show an error message to the user near the submit button informing them of the error, and the form is not sent. (checked).
-function in the validation.js.
-Google chrome is used for this activity.
Thank you.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -319,7 +319,6 @@ <h4>Let's chat!</h4>
                 </p>
```
Comment (SajjadAhmad14) [inline index.html:319]: Please add linter for HTML, CSS, and JavaScript from [this](https://github.com/microverseinc/linters-config/tree/master/html-css-js) linter config.

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi Aku Khan, 
 ### You have been doing a tremendous job so far :ok_hand:

## To Highlight 🎉
GitHub flow is followed. :heavy_check_mark:
README is professional. :heavy_check_mark:
## To Improve 🙏

- [ ] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity!

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by SajjadAhmad14 — APPROVED** [review]
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:

### Hi Aku Khan, 
## Congratulations 🎉 Thank you for implementing changes suggested previously 🙏

## Happy Coding 👏👏👏
#### Please leave any questions or comments in the PR if you have any doubts or unclarity!
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Comment by aakbarkhan** [comment]
Thank you all.

**Pull request merged**